sale_margin_percentage/security/sale_margin_percentage_security.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Shouldn't be noupdate?

Reply:
Why should it be 'noupdate=true'?

Reply:
Data is usually noupdate so it may be edited functionally, otherwise changes would be lost next time the module is updated.

Reply:
ok, already apply the change.